src/server.prod.js

```diff
@@ -49,9 +57,13 @@ class ProdServer {
     const nockPath = this.config.nockPath;
     const accessLogs = this.config.accessLogs || 'clf';
     const renderFn = this.getRenderFunction();
+    if (!isUrl(serverPublicPath)) {
+      app.use(serverPublicPath, express.static(serverOutputPath));
+    }
+    if (!isUrl(clientPublicPath)) {
+      app.use(clientPublicPath, express.static(clientOutputPath));
+    }
```

Comment:
This maybe should also be possible in the server.dev.js file.

Reply:
Also a documentation of this function would be awesome.

Reply:
The dev server just uses the given public path or URL, no need to change anything.

Reply:
You are absolutely correct.